.eslintignore

```diff
@@ -7,6 +7,7 @@ tests/files/with-syntax-error
 tests/files/just-json-files/invalid.json
 tests/files/typescript-d-ts/
 resolvers/webpack/test/files
+vendors
```

Comment:
Vendoring is an obsolete and harmful practice, so if get-tsconfig doesn't support the node versions we support (by transpiling down to them, testing on them, and explicitly declaring engines for them), and is unwilling to do so, then this is a nonstarter.

Are they willing to do so?

Reply:
Maybe I can fork a down-leveled version?

Reply:
That doesn't seem much better than just sticking with the older tsconfig-paths that we have.

Reply:
get-tsconfig replaces tsconfig-paths + TypeScript at the same time, and it improves the performance a lot.

Reply:
And also `jsconfig.json` is supported without breaking.

Reply:
And how is vendors worse then pin old version of `tsconfig-paths`? A pinned version is just same as vendors, and we can also try to find a better way to downlevel.

Reply:
Including minified code in a package that is not generated by the repo itself causes tons of false positives on security scanning tools, and is a wonderful way to create a future problem when someone sneaks malicious code into it.

We are simply not going to vendor; it's not an acceptable choice.

Pinning an old version is 100% fine; it's a separate dependency, it's immutable. It is in no way the same as vendoring.

Reply:
It's open source, you're free to do that.

Reply:
Also it maybe time to consider a breaking change instead for this project. Then a lot of legacy dependency `issues` can be resolved.

Reply:
I don't see sufficient value in doing so.

Reply:
requiring `typescript` inside is not effective method for performance.